apps/frontend/src/app/legal/legal/legal.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="container mt-5 mb-5">
+  <h2>Privacy Policy</h2>
+  <p class="text-muted">Last updated: August 19, 2023</p>
+  <p>This Privacy Policy describes the use and disclosure of your information when you are using this service.
```

Comment:
Can you put a newline after all the ￼tags?

Reply:
Done in [565c3be](https://github.com/Morphclue/apollusia/pull/91/commits/565c3bec496743ce1a00c500fd253097ebbdb5a6).